CONTRIBUTING.md

```diff
@@ -80,9 +80,26 @@ withdrawn it will be moved to an archive folder.
 
 ## Manual checking of the PDF documents and Continuous Integration
 
-To check the outcome of your changes, run the `tools/rst2pdf/generate-pdfs.sh`
-script. To install the (python) prerequisites for the script, run the
-`tools/rst2pdf/install.sh` script.
+To check the outcome of your changes, use the `tools/rst2pdf/generate-pdfs.sh`
+script. To install the (python) prerequisites for the script, do the following.
+
+Create and activate Python virtual environment (you will need to use Python 3.x):
```

Comment:
We might be able to get rid of the `(you will need to use Python 3.x)` if we use `python3` and `pip3` explicitly.